content/en/tracing/setup_overview/setup/php.md

```diff
@@ -134,7 +134,7 @@ Automatic instrumentation captures:
 * Unhandled exceptions, including stacktraces if available
 * A total count of traces (for example, web requests) flowing through the system
 
-**Note**: If your application does not use Composer nor an autoloader registered with `spl_autoload_register()`, set the environment variable, `DD_TRACE_NO_AUTOLOADER=true`, to enable automatic instrumentation.
+**Note**: If your application does not use Composer nor an autoloader registered with `spl_autoload_register()`, set the environment variable, `DD_TRACE_NO_AUTOLOADER=1`, to enable automatic instrumentation.
```

Comment:
I was curious what this is, but that env seems to be nowhere referenced in our current code? Sounds obsolete?